packages/gateway/src/features/search/type-resolvers.ts

```diff
@@ -420,6 +420,9 @@ export const searchTypeResolvers: GQLResolver = {
       _,
       { dataSources }
     ) {
+      // eventLocationId is not present for a record created by users
```

Comment:
```suggestion
      // eventLocationId is not present if the encounter wasn't in a health facility
```
maybe this is more accurate? as it can be user created I think